tests/src/python/qgis_wrapped_server.py

```diff
@@ -39,6 +39,9 @@
 
   ?MAP=/path/to/projects.qgs&SERVICE=XYZ&X=1&Y=0&Z=1&LAYERS=world
 
+Note that multi threading in QGIS server is not officially supported and
```

Comment:
All other places in the codebase use a single word "multithreading"

```suggestion
Note that multithreading in QGIS server is not officially supported and
```

Reply:
See also https://github.com/qgis/QGIS/pull/9127